### PR TITLE
SF4 compatibility

### PR DESCRIPTION
**Pull request by alanpoulain**
Not much to say :smile:

### CODE REVIEW

**Review by vincentchalamon — APPROVED** [review]
Waiting for CI to be green, then :+1:

**Comment by Simperfit** [comment]
CI does not seems to be green

**Comment by gorghoa** [comment]
thanks @Simperfit :rofl:

File: features/scenario_state.feature [hunk 1]
```diff
@@ -12,7 +12,7 @@ Feature: Scenario shared state
     When I run "behat --no-colors features/donkeys.feature"
     Then it should fail with:
     """
-    [Behat\Testwork\Argument\Exception\UnknownParameterValueException]
+    Can not find a matching value for an argument
```
Comment (alanpoulain) [inline features/scenario_state.feature:15]: Output differs from Behat >= 3.1.0.

**Comment by vincentchalamon** [comment]
Thanks @alanpoulain

File: features/scenario_state.feature (hunk 1 above)
Comment (gorghoa) [reply features/scenario_state.feature:15]: nice, thanks @alanpoulain :)

**Pull request merged**